pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.black]
+line-length = 120
+target-version = ['py38']
+include = '\.pyi?$'
+exclude = '''
+/(
+    \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | _build
+  | build
+  | dist
+)/
+'''
+
+[tool.mypy]
+python_version = "3.8"
```

Comment:
Since we build with multiples images and python version, I would remove that one and leave that up to the platform